sphinx.conf

```diff
@@ -262,7 +262,7 @@ searchd
  read_timeout  = 5
  max_children  = 30
  pid_file   = /var/run/sphinx/searchd.pid
- max_matches   = 2000
+ max_matches   = 1000
```

Comment:
No.  I did not make that change.



-- kevin carter
-- programmer analyst
-- wgbh educational foundation
-- 1 guest street
-- boston, ma, usa, 02135
-- 6173005923 v/vm/fax
-- www.wgbh.org

-- WGBH:  pioneering public education since 1951 with content and
technology to enlighten and enable.
________________________________
From: afred [notifications@github.com]
Sent: Wednesday, January 21, 2015 3:51 PM
To: WGBH/AMS
Cc: Kevin Carter
Subject: Re: [AMS] Changes max_matches from 2000 to 1000. (#1)


In sphinx.conf<https://github.com/WGBH/AMS/pull/1#discussion_r23333268>:

> @@ -262,7 +262,7 @@ searchd
>   read_timeout  = 5
>   max_children  = 30
>   pid_file   = /var/run/sphinx/searchd.pid
> - max_matches   = 2000
> + max_matches   = 1000


@foo4thought<https://github.com/foo4thought> Did you make this change?

—
Reply to this email directly or view it on GitHub<https://github.com/WGBH/AMS/pull/1/files#r23333268>.